examples/2d_examples/FlowPastConstantHeatFluxCylinderCase/flow_past_constant_heat_flux_cylinder.py

```diff
@@ -0,0 +1,372 @@
+
+    # Initialize fixed cylinder (elastica rigid body) with direction along Z
+    x_cm = 0.5 * x_range  # 10.5 * cyl_radius  # 0.2 * x_range #2.5 * cyl_radius
+    y_cm = 0.5 * y_range
+    start = np.array([x_cm, y_cm, 0.0])
+    direction = np.array([0.0, 0.0, 1.0])
+    normal = np.array([1.0, 0.0, 0.0])
+    base_length = 1.0
+    density = 1e3
+    cylinder = ea.Cylinder(start, direction, normal, base_length, cyl_radius, density)
+
+    # Since the cylinder is fixed, we dont add it to pyelastica simulator,
+    # and directly use it for setting up the flow interactor.
+
+    # ==================FLOW-BODY COMMUNICATOR SETUP START======
+    num_lag_nodes = int(grid_size_x // 5 * 1.5)
+
+    cylinder_flow_interactor = sps.RigidBodyFlowInteraction(
+        rigid_body=cylinder,
+        eul_grid_forcing_field=flow_sim.eul_grid_forcing_field,
+        eul_grid_velocity_field=flow_sim.velocity_field,
+        virtual_boundary_stiffness_coeff=coupling_stiffness,
+        virtual_boundary_damping_coeff=coupling_damping,
+        dx=flow_sim.dx,
+        grid_dim=grid_dim,
+        real_t=real_t,
+        forcing_grid_cls=sps.CircularCylinderForcingGrid,
+        num_forcing_points=num_lag_nodes,
+    )
+
+    virtual_thermal_layer_interactor = sps.RigidBodyFlowInteraction(
+        rigid_body=cylinder,
+        eul_grid_forcing_field=thermal_sim.eul_grid_forcing_field,
+        eul_grid_velocity_field=thermal_sim.primary_field,
+        virtual_boundary_stiffness_coeff=coupling_stiffness * 0,
+        virtual_boundary_damping_coeff=coupling_damping * 0,
+        dx=thermal_sim.dx,
+        grid_dim=grid_dim,
+        real_t=real_t,
+        field_type="scalar",
+        forcing_grid_cls=sps.CircularCylinderVirtualLayerTemperatureForcingGrid,
+        num_forcing_points=num_lag_nodes,
+        eul_dx=3 * thermal_sim.dx,
+    )
+
+    isoflux_cylinder_thermal_interactor = sps.RigidBodyFlowInteraction(
+        rigid_body=cylinder,
+        eul_grid_forcing_field=thermal_sim.eul_grid_forcing_field,
+        eul_grid_velocity_field=thermal_sim.primary_field,
+        virtual_boundary_stiffness_coeff=coupling_stiffness,
+        virtual_boundary_damping_coeff=coupling_damping,
+        dx=thermal_sim.dx,
+        grid_dim=grid_dim,
+        real_t=real_t,
+        field_type="scalar",
+        forcing_grid_cls=sps.CircularCylinderIndirectNeumannConditionForcingGrid,
+        num_forcing_points=num_lag_nodes,
+        virtual_layer_interactor=virtual_thermal_layer_interactor,
+        heat_flux=heat_flux,
+        eul_dx=3 * thermal_sim.dx,
+    )
+
+    # ==================FLOW-BODY COMMUNICATOR SETUP END======
+
+    # iterate
+    timescale = cyl_radius / velocity_scale
+    final_time = nondim_final_time * timescale  # dimensional end time
+    foto_timer = 0.0
+    foto_timer_limit = final_time / 50
+
+    data_timer = 0.0
+    data_timer_limit = 0.25 * timescale
+    drag_coeffs_time = []
+    drag_coeffs = []
+    nusslet_number_time = []
```

Comment:
Should be 'nusselt' instead of 'nusslet' if you are referring to the Nusselt number. There seems to be 57 occurrences across the files.